backend/app/models.py

```diff
@@ -1,6 +1,39 @@
 """
-Models for the ***** app.
+Models for the gender analysis web app.
 """
-# from django.db import models
+from django.db import models
+from .fields import LowercaseCharField
 
-# no models yet -- write me!
+
+class Pronoun(models.Model):
+    """
+    A model that allows users to define an individual pronoun and its type
+    (e.g. subject, object, reflexive, etc). Pronouns are case-insensitive and will be
+    converted to lowercase.
+    """
+    PRONOUN_TYPES = [
+        ('subj', 'Subject'),
+        ('obj', 'Object'),
+        ('pos_det', 'Possessive determiner'),
+        ('pos_pro', 'Possessive pronoun'),
+        ('reflex', 'Reflexive'),
+    ]
+
+    pronoun = LowercaseCharField(max_length=40)
```

Comment:
This is minor and subjective, but I wonder if this `pronoun` field should be named something other than `pronoun`, since that's also the name of the model. I'm thinking that there might be scenarios that end up looking like this:
```
pronoun = Pronoun(...)
pronoun = pronoun.pronoun
```

Maybe we could call this `identifier` or something?